src/redist/targets/GenerateMSIs.targets

```diff
@@ -318,8 +318,11 @@
 
     <!-- Choose "latest" template MSI to go in bundle. -->
     <ItemGroup>
+      <!-- While we don't have 8.0 templates available (need SDK update, choose the 7.0 templates -->
+      <!-- <LatestTemplateInstallerComponent Include="@(TemplatesMsiComponent)"
+                                        Condition="'%(TemplatesMajorMinorVersion)' == '$(MajorMinorVersion)'"/> -->
       <LatestTemplateInstallerComponent Include="@(TemplatesMsiComponent)"
-                                        Condition="'%(TemplatesMajorMinorVersion)' == '$(MajorMinorVersion)'"/>
+                                         Condition="'%(TemplatesMajorMinorVersion)' == '7.0'"/>
```

Comment:
We'll have to change this again once we have 8.0 templates?

Reply:
Yup, that's what we did in 7.0. We hardcoded it to the 6.0 versions until we had templates available and then fixed it later. @vlada-shubina for visibility.

Reply:
I ended up fixing it in Jan: https://github.com/dotnet/installer/pull/13095

Reply:
I guess this needs all the templates update (aspnet, winforms, wpf, test), not only those we manage.
I plan to do the templates that are now in SDK repo next week.